core/src/main/java/com/oxeanbits/forecastchart/core/util/DateFormatter.kt

```diff
@@ -4,9 +4,10 @@ import java.text.SimpleDateFormat
 import java.util.Date
 
 object DateFormatter{
+    const val ONE_DAY = 86400
     fun timestampToDate(timestamp: Long): String{
         val mDate = Date()
-        mDate.time = (timestamp.times(1000))
+        mDate.time = timestamp.times(1000) + ONE_DAY
```

Comment:
Lets check if the date coming in the array will not change regardless the timezone of the user.